source/change-streams/tests/change-streams.yml

```diff
@@ -0,0 +1,339 @@
+collection_name: &collection_name "test"
+database_name: &database_name "change-stream-tests"
+collection2_name: &collection2_name "test2"
+database2_name: &database2_name "change-stream-tests-2"
+tests:
+  -
+    description: "$changeStream must be the first stage in a change stream pipeline sent to the server"
+    minServerVersion: "3.6.0"
+    target: collection
+    topology: 
+      - replicaset
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: insertOne
+        arguments:
+          document:
+            x: 1
+    expectations:
+      - 
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              - 
+                $changeStream:
+                  fullDocument: default
+          command_name: aggregate
+          database_name: *database_name
+    result:
+      success: []
```

Comment:
Shouldn't the `insertOne` operation be observed here? Or is omitted because the test only concerns itself with the `aggregate` command structure in the APM event. I suppose that would explain why the subsequent "The server returns change stream responses in the specified server response format" test ignores the APM format.

Reply:
Yes, since the test is only meant to test the `aggregate` command structure, that is the only event listed. IMO it is best to test only as much as necessary to assert the condition.